src/kvstore/EventListner.h

```diff
@@ -13,31 +13,243 @@ namespace kvstore {
 
 class EventListener : public rocksdb::EventListener {
 public:
-    void OnStallConditionsChanged(const rocksdb::WriteStallInfo& info) override {
-        LOG(INFO) << "Rocksdb write stall condition changed, column family: " << info.cf_name
-                  << ", pre " << static_cast<int32_t>(info.condition.prev)
-                  << ", cur " << static_cast<int32_t>(info.condition.cur);
-    }
-
+    // A callback function to RocksDB which will be called before a RocksDB starts to compact.
     void OnCompactionBegin(rocksdb::DB*, const rocksdb::CompactionJobInfo& info) override {
-        LOG(INFO) << "Rocksdb compact column family: " << info.cf_name
-                  << " because of " << static_cast<int32_t>(info.compaction_reason)
+        LOG(INFO) << "Rocksdb start completion column family: " << info.cf_name
+                  << " because of " << compactionReasonString(info.compaction_reason)
                   << ", status: " << info.status.ToString()
                   << ", compacted " << info.input_files.size()
                   << " files into " << info.output_files.size()
                   << ", base level is " << info.base_input_level
                   << ", output level is " << info.output_level;
     }
 
+    // A callback function for RocksDB which will be called
+    // whenever a registered RocksDB compacts a file.
     void OnCompactionCompleted(rocksdb::DB*, const rocksdb::CompactionJobInfo& info) override {
-        LOG(INFO) << "Rocksdb compact column family: " << info.cf_name
-                  << " because of " << static_cast<int32_t>(info.compaction_reason)
+        LOG(INFO) << "Rocksdb compaction completed column family: " << info.cf_name
+                  << " because of " << compactionReasonString(info.compaction_reason)
                   << ", status: " << info.status.ToString()
                   << ", compacted " << info.input_files.size()
                   << " files into " << info.output_files.size()
                   << ", base level is " << info.base_input_level
                   << ", output level is " << info.output_level;
     }
+
+    // A callback function to RocksDB which will be called
+    // before a RocksDB starts to flush memtables.
+    void OnFlushBegin(rocksdb::DB*, const rocksdb::FlushJobInfo& info) override {
+        LOG(INFO) << "Rocksdb start flush column family: " << info.cf_name
```

Comment:
VLOG is enough for new added EventListener?

Reply:
Not enough. `Flush`  and `Ingest` should logged in INFO Level.